app.js

```diff
@@ -28,5 +28,6 @@ db.sequelize
 
 app.use('/', indexRouter)
 app.use('/users', usersRouter)
+app.use('/doc', express.static('apidoc'));
```

Comment:
Pas de semicolon stp (ESLint)
```suggestion
app.use('/doc', express.static('apidoc'))
```

Reply:
ça va être dur ça par contre... mais je vais essayer...